xcover/xcover.py

```diff
@@ -54,7 +74,15 @@ def call_partner_endpoint(self, method, url, payload=None, **kwargs):
         full_url = urljoin(f"partners/{self.partner_code}/", url)
 
         # Call server
-        response = self.call(method, full_url, payload=payload, **kwargs)
+        if method in {"POST", "PUT", "PATCH"}:
+            headers = kwargs.pop("headers", {})
+            headers.setdefault("x-idempotency-key", str(uuid4()))
```

Comment:
We don't support idempotency-key for quote operation. 

Shall we exclude it here as well to make it obvious it is not supported?

Reply:
@tyomo4ka do you mean in XCore we don't support idempotency-key for quote operation? If so, I think it's better to exclude it here.